drain-service/drain3/drain.py

```diff
@@ -10,16 +10,20 @@
 
 
 class LogCluster:
-    __slots__ = ["log_template_tokens", "cluster_id", "size"]
+    __slots__ = ["log_template_tokens", "cluster_id", "anomaly_level","size"]
 
-    def __init__(self, log_template_tokens: list, cluster_id: int):
+    def __init__(self, log_template_tokens: list, cluster_id: int, anomaly_level="Normal"):
```

Comment:
what does `anomaly_level` do?

Reply:
It's now going to be a class variable so when the DRAIN control plane inferencing service attempts to match logs with a given template, it'll be able to get the anomaly level of any matched templates.